test/qa/test_connect.py

```diff
@@ -450,3 +452,25 @@ def test_status_connected(tech, proto, obfuscated):
         assert time_passed - 1 <= time_connected <= time_passed + 1
 
     disconnect_base_test()
+
+
+@pytest.mark.parametrize(("tech", "proto", "obfuscated"), lib.STANDARD_TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(40)
+def test_connect_to_virtual_server(tech, proto, obfuscated):
+    lib.set_technology_and_protocol(tech, proto, obfuscated)
+    sh.nordvpn.set("virtual-location", "on")
+    output = sh.nordvpn.countries().stdout.decode("utf-8")
+
+    # This pattern captures all substring starting with \x1b\[94m[ that are single words. It should capture all of the
+    # virtual server names, as in the terminal output they are colored blue.
+    pattern = r"\x1b\[94m\w+\x1b\[0m"
```

Comment:
It might be nice to have also a test that there is no color when `_tty_out=False` when calling the command.